libuavcan_drivers/linux/scripts/uavcan_add_vcan

```diff
@@ -19,11 +19,14 @@ if [ "$1" == '--help' ] || [ "$1" == '-h' ]; then echo "$HELP"; exit; fi
 
 IFACE="$1"
 
-if [ $(ifconfig -a | grep -c "^$IFACE") -eq "1" ]; then
-    ifconfig $IFACE up
+ip link show $IFACE > /dev/null
+if [ $? == 0 ]; then
+    ip link set up $IFACE
```

Comment:
Can we not just say `if ! ip link show $IFACE > /dev/null; then` instead of the two lines above?